snapcraft/plugins/ament.py

```diff
@@ -105,6 +105,7 @@ def __init__(self, name, options, project):
             version=self.options.version,
             bootstrap_path=self._bootstrap_dir,
             ubuntu_sources=self.PLUGIN_STAGE_SOURCES,
+            ubuntu_keyrings=self.PLUGIN_STAGE_SOURCES,
```

Comment:
This isn't quite the [commit I've verified](https://github.com/artivis/snapcraft/commit/c1edb60e69540e7f380b3a757e6439ea5695d0f8). It needs the keyring.

Reply:
My bad I'm messed-up pushing an outdated commit.